sematic/plugins/metrics_storage/sql/models/metric_value.py

```diff
@@ -41,3 +45,6 @@ class MetricValue(Base):
     created_at: datetime.datetime = Column(
         types.DateTime(), nullable=False, default=datetime.datetime.utcnow
     )
+
+Index("metric_values_id_time_idx", MetricValue.metric_id, MetricValue.metric_time.desc()),
```

Comment:
these can't be added via `__table_args__` like above, because of the `desc` sort on `metric_time`.  to specify the sort, you need a reference to `MetricValue`, just the string `metric_values.metric_time` will not suffice.